libs/core/src/lib/utils/pipes/search-highlight.pipe.ts

```diff
@@ -5,6 +5,8 @@ import { Pipe, PipeTransform } from '@angular/core';
 })
 export class SearchHighlightPipe implements PipeTransform {
     transform(value: string, args: string, active: boolean = true, includeSpans: boolean = false): string {
+        value = value.replace(/</g, '&lt;');
```

Comment:
Since we use lodash anyways, why not [utilize it escape](https://lodash.com/docs/#escape)?